dbtmetabase/parsers/dbt_folder.py

```diff
@@ -0,0 +1,196 @@
+            list -- List of dbt models in Metabase-friendly format.
+        """
+
+        mb_models: List[MetabaseModel] = []
+
+        for path in (Path(self.project_path) / "models").rglob("*.yml"):
+            logging.info("Processing model: %s", path)
+            with open(path, "r") as stream:
+                schema_file = yaml.safe_load(stream)
+                if schema_file is None:
+                    logging.warn("Skipping empty or invalid YAML: %s", path)
+                    continue
+                for model in schema_file.get("models", []):
+                    name = model.get("identifier", model["name"])
+                    if "identifier" in model:
+                        self.catch_aliases[name] = model["name"]
+                    logging.info("Model: %s", name)
+                    if (not includes or name in includes) and (name not in excludes):
+                        mb_models.append(
+                            self._read_model(model, schema, include_tags=include_tags)
+                        )
+                for source in schema_file.get("sources", []):
+                    for model in source.get("tables", []):
+                        name = model.get("identifier", model["name"])
+                        if "identifier" in model:
+                            self.catch_aliases[name] = model["name"]
+                        logging.info("Source: %s", name)
+                        if (not includes or name in includes) and (
+                            name not in excludes
+                        ):
+                            mb_models.append(
+                                self._read_model(
+                                    model, schema, include_tags=include_tags
+                                )
+                            )
+
+        return mb_models
+
+    def _read_model(self, model: dict, schema: str, include_tags=True) -> MetabaseModel:
+        """Reads one dbt model in Metabase-friendly format.
+
+        Arguments:
+            model {dict} -- One dbt model to read.
+
+        Returns:
+            dict -- One dbt model in Metabase-friendly format.
+        """
+
+        mb_columns: List[MetabaseColumn] = []
+
+        for column in model.get("columns", []):
+            mb_columns.append(self._read_column(column, schema))
+
+        description = model.get("description")
+
+        if include_tags:
+            tags = model.get("tags")
+            if tags:
+                tags = ", ".join(tags)
+                if description != "":
+                    description += "\n\n"
+                description += f"Tags: {tags}"
+
+        return MetabaseModel(
+            # We are implicitly complying with aliases by doing this
+            name=model.get("identifier", model["name"]).upper(),
+            schema=schema,
+            description=description,
+            columns=mb_columns,
+        )
+
+    def _read_column(self, column: dict, schema: str) -> MetabaseColumn:
```

Comment:
Another example of the above. `Mapping` as a type hint could be preferred over `dict`.